px-context-browser.html

```diff
@@ -867,13 +1060,29 @@
     },
 
     _isItemSelectable(item) {
-      return this._assetGraph.isSelectable(item);
+      return item && this._assetGraph.isSelectable(item);
+    },
+
+    _isItemFavoritable(item) {
+      return item && this._assetGraph.isSelectable(item);
     },
 
     _isNotNull(val) {
       return (typeof val !== 'undefined' && val !== null);
     },
 
+    _showHeader(active) {
```

Comment:
Why do we need this? Can't we just use `_isNotNull(active)` in the template, instead of wrapping another function around it that just calls that `_isNotNull` function?

Reply:
:+1: that was leftover from when it was checking more than just _isNotNull